text2sql-frontend/src/components/Settings/OpenAIKeyPopup.tsx

```diff
@@ -56,6 +56,19 @@ export function OpenAIKeyPopup() {
           onKeyUp={handleKeyPress}
         />
       </AlertBody>
+      <AlertDescription>
+        <p className="text-xs">
+          * Your API key must have{" "}
```

Comment:
I would replace this with a call to action:

"Please update your API key with <full permissions> to use DataLine".

As it is now, some users might thing ah ok I dont HAVE to do this.

Reply:
Done